src/store/modules/brushModule.js

```diff
@@ -66,34 +57,30 @@ const mutations = {
       state.visibleSegmentations[enabledElementUID].push(true);
     }
   },
-  SET_ELEMENT_BRUSH_VISIBILITY: (enabledElementUID, segIndex, visible = true) => {
+  setBrushVisibilityForElement: (
+    enabledElementUID,
+    segIndex,
+    visible = true
+  ) => {
     if (!state.visibleSegmentations[enabledElementUID]) {
       state.imageBitmapCache[enabledElementUID] = [];
     }
 
     state.visibleSegmentations[enabledElementUID][segIndex] = visible;
   },
-  SET_ELEMENT_IMAGE_BITMAP_CACHE: (enabledElementUID, segIndex, imageBitmap) => {
+  setImageBitmapCacheForElement: (enabledElementUID, segIndex, imageBitmap) => {
```

Comment:
Since we are going back to camelCase, we can safely turn the new-cap eslint rule on, as it should now only be for constructors.